### ISSUE TITLE
Run ActivitySim with Beam using the new Merging feature

### ISSUE THREAD

**Issue by dimaopen** [LBNL-UCB-STI/beam#3209]
Merging feature: #3192

**Comment by dimaopen** [comment]
I tested the ActivitySim plans with the Last Run Plan Merger feature. I run the initial ActivitySim plans as they are. Then I modified the plans: all Car modes changed to Walk. And I have run 2 subsequent runs with 20% of plan merging.  Here's what we have

1. The realized modes of the initial plans
![1 realizedModeChoice](https://user-images.githubusercontent.com/1764256/122362213-9e9f9280-cf60-11eb-9627-2f882e668b5f.png)
2. The second run which took the last BEAM output plans (from item 1) and merged the modified ActivitySim plans (20%)
![2 realizedModeChoice](https://user-images.githubusercontent.com/1764256/122362242-a4957380-cf60-11eb-9da7-759c3b34886e.png)
3. The third run took the last output (item 2) and merged the modified ActivitySim plans (20%).
![3 realizedModeChoice](https://user-images.githubusercontent.com/1764256/122362269-a9f2be00-cf60-11eb-9b9a-973613bb8678.png)
